gnomad/resources/config.py

```diff
@@ -0,0 +1,47 @@
+"""Configuration for loading resources."""
+
+import typing
+from enum import Enum
+
+
+class GnomadPublicResourceSource(Enum):
```

Comment:
Grouping related constants in an enum lets us use the type system to represent that relationship. With multiple string constants, it's up to the developer to remember that they are related. We can also use more accurate type annotations, annotating an argument as an enum type rather than annotating it as a string with accepted values in a docstring. That makes static analysis tools, IDE autocompletion, etc. more effective. Granted though, some of those same benefits could also be realized with [literal types](https://www.python.org/dev/peps/pep-0586/) once we can use Python 3.8 (not sure what version of Python is on Dataproc clusters started with `hailctl`).

Reply:
OK, so possibly a discussion moving forward, thank you!